data/constructron-rocket-powered.lua

```diff
@@ -80,37 +80,20 @@ local constructron_item = {
 }
 
 local constructron_recipe = {
-    type = "recipe",
-    name = "constructron-rocket-powered",
-    enabled = false,
-    ingredients = {
-        {"raw-fish", 1},
-        {"rocket-control-unit", 16},
-        {"low-density-structure", 150},
-        {"effectivity-module-3", 2},
-        {"rocket-launcher", 4},
-        {"fusion-reactor-equipment", 2},
-        {"exoskeleton-equipment", 4},
-        {"radar", 2},
-        {"se-cargo-rocket-section", 200}
-    },
-    result = "constructron-rocket-powered",
-    result_count = 1,
-    energy = 1
+  type = "recipe",
+  name = "constructron-rocket-powered",
+  enabled = false,
+  ingredients = {
+    {"constructron", 1},
+    {"se-cargo-rocket-section", 200}
+  },
+  result = "constructron-rocket-powered",
+  result_count = 1,
+  energy = 1
 }
 
-local constructron_easy_recipe = {
-    type = "recipe",
-    name = "constructron-rocket-powered",
-    enabled = false,
-    ingredients = {
-        {"spidertron", 1},
-        {"se-cargo-rocket-section", 200}
-    },
-    result = "constructron-rocket-powered",
-    result_count = 1,
-    energy = 1
-}
+-- both easy and normal recipes add rocket sections to a normal constructron
+local constructron_easy_recipe = constructron_recipe
```

Comment:
so we effectively don't have a easy/normal recipe distinction anymore?

not too sure what to think of this.

Reply:
We do, since the rocket ctron is based on the ctron. If you have hard selected, the base ctron will have that recipe, and then the rocket ctron will get that automatically. I didn't see a difference other than the rocket parts between the rocket ctron and the normal ctron. Also, the hard recipe is quite different than the SE spider recipe. The fusion-reactor-equipment is much further down the tech tree in SE. Normal spider in SE has a very different recipe than vanilla spider, using some SE ingredients.